server/apps/postgres-app/src/test/java/org/apache/james/PostgresJamesServerWithAuthenticatedDatabaseSqlValidationTest.java

```diff
@@ -35,7 +37,7 @@ class PostgresJamesServerWithAuthenticatedDatabaseSqlValidationTest extends Post
             .build())
         .server(configuration -> PostgresJamesServerMain.createServer(configuration)
             .overrideWith(new TestJPAConfigurationModuleWithSqlValidation.WithDatabaseAuthentication()))
-        .extension(PostgresExtension.empty())
+        .extension(PostgresExtension.withRowLevelSecurity(PostgresModule.aggregateModules(PostgresDomainModule.MODULE)))
```

Comment:
:red_circle: 

The Postgres module should come from the james server being launched and not duplicated in each test.

Reply:
I know cf previous comment, was just a trick... working on it

Reply:
Having a race issue between tables init creation and domain init conf...